sysfs/system_cpu.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sysfs
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/prometheus/procfs/internal/util"
+)
+
+// SystemCPUCpufreq contains stats from devices/system/cpu/cpu[0-9]*/cpufreq/...
+type SystemCPUCpufreq struct {
```

Comment:
Nit: I would go with something like `SystemCPUFrequency` as that's more human friendly than the Linux directory name.

Reply:
I prefer the literal Linux names most of the time.

Reply:
tbh, I don't like either of these names. The name should convey what this is about. So maybe (System)CPUStats?

Reply:
I strongly prefer to use the names we get from Linux, this is the least surprising mapping between the library and the kernel.

Reply:
I get where you're coming from but if you just look at the code without knowing the linux names it's just hard to read. Let's append at least `Stats` or something because this struct isn't a "frequency" then?